### PR TITLE
[API-5517] Add recommendation API support

### PR DESCRIPTION
**Pull request by davidhuang01**
Add recommendation API support.

### CODE REVIEW

File: src/main/java/com/visenze/productsearch/ProductSearchResponse.java [hunk 1]
```diff
@@ -150,6 +152,16 @@
     @JsonProperty("query_sys_meta")
     private Map<String, String> querySysMeta;
 
+    @JsonProperty("explanation")
+    @JsonInclude(JsonInclude.Include.NON_EMPTY)
+    private Map<String, ViJsonAny> explanation;
```
Comment (thehung111) [inline src/main/java/com/visenze/productsearch/ProductSearchResponse.java:157]: i think we have not confirmed what to put for explanation yet, might as well remove this for now.

**Review by thehung111 — APPROVED** [review]
LGTM

**Comment by thehung111** [comment]
@davidhuang01 can you bump the version ? thank you

File: src/main/java/com/visenze/productsearch/ProductSearchResponse.java (hunk 1 above)
Comment (davidhuang01) [reply src/main/java/com/visenze/productsearch/ProductSearchResponse.java:157]: OK, I'll remove this for now.

**Comment by davidhuang01** [comment]
> @davidhuang01 can you bump the version ? thank you

Is 1.11.1 good to you?

**Comment by thehung111** [comment]
> > @davidhuang01 can you bump the version ? thank you
> 
> Is 1.11.1 good to you?

yes it is fine. Thank you.

**Pull request merged**